clouds/cli/cli.py

```diff
@@ -0,0 +1,106 @@
+import os
+import click
+import shutil
+from simple_logger.logger import get_logger
+from clouds.cli.aws.aws_cli import clean_aws_resources
+from pyhelper_utils.runners import function_runner_with_pdb
+from clouds.cli.microsoft_azure.microsoft_azure_cli import nuke_all_azure_resources
+from clouds.aws.aws_utils import set_and_verify_aws_credentials, aws_region_names
+
+LOGGER = get_logger(name="nuke-cli")
+
+
+@click.group()
+def aws() -> None:
```

Comment:
should get `--pdb` no?

Reply:
Don't understand the question.
If we want to add `--pdb` we should add it as `click.option` to `aws.command` and `azure.command`

Reply:
the code call `function_runner_with_pdb(func=cloud_cli)`, how do you pass pdb to it?

Reply:
check `function_runner_with_pdb` docstring

Reply:
We can't pass a `click.option` to a `click.CommandCollection` so the current approach wouldn't work.
We can alternatively use aws/azure as click.option (e.g. `--cloud-provider`) or use separate CLIs or any other you can think of.
Let me know wyt @rnetser @myakove

Reply:
you should use `ctx` to pass it between the clis